package.json

```diff
@@ -4,13 +4,14 @@
   "private": true,
   "dependencies": {
     "@beyondidentity/bi-sdk-js": "^1.0.10",
-    "@docusaurus/core": "^2.2.0",
-    "@docusaurus/plugin-debug": "^2.2.0",
-    "@docusaurus/plugin-google-gtag": "^2.2.0",
-    "@docusaurus/plugin-sitemap": "^2.2.0",
-    "@docusaurus/remark-plugin-npm2yarn": "^2.2.0",
-    "@docusaurus/theme-classic": "^2.2.0",
-    "@docusaurus/theme-search-algolia": "^2.2.0",
+    "@docusaurus/core": "^2.3.0",
```

Comment:
thanks for this!

Reply:
np! I wanted to be able to link to the sdk page to a specific tab, which was literally just released! 😄